dist/commands/head.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var interfacer = require('./../util/interfacer');
+var fs = require('fs');
+var fsAutocomplete = require('vorpal-autocomplete-fs');
+
+var head = {
+  exec: function exec(args, options) {
+    var self = this;
+    options = options || {};
+
+    options.argsType = args.stdin === undefined ? 'files' : 'stdin';
+    options.n = options.n === undefined ? 10 : options.n;
+
+    if (options.n < 1) {
+      self.log('Option n must be a positive integer.');
```

Comment:
Linux `head` actually takes a negative argument as well. `head -n -1` means "output the all the lines of the file, except the last line," `-n -3` means "all the lines except for the last 3 lines," etc.

Also, `-n 0` also works, it just outputs nothing, with status `0` (only if it can read the file, otherwise it fails).